CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add events correlation engine plugin ([#6854](https://github.com/opensearch-project/OpenSearch/issues/6854))
 - Add support for ignoring missing Javadoc on generated code using annotation ([#7604](https://github.com/opensearch-project/OpenSearch/pull/7604))
+- Added server version as response header [#6583](https://github.com/opensearch-project/OpenSearch/issues/6583)
+- Implement concurrent aggregations support without profile option ([#7514](https://github.com/opensearch-project/OpenSearch/pull/7514))
+- Add dynamic index and cluster setting for concurrent segment search ([#7956](https://github.com/opensearch-project/OpenSearch/pull/7956))
```

Comment:
Why are we adding these here?

Reply:
They must of got caught in a merge conflict

Reply:
Can you rebase with latest `main`? That will remove these additions.

Reply:
Rebased so only has my change in change log

Reply:
Do we still need @dblock to check the comment about the implementation?

Reply:
Thanks @Sam-ScottLogic! Looks like you have rebase again since https://github.com/opensearch-project/OpenSearch/pull/8290 got merged which resolves the failing gradle check on latest `main`. 
Yes, let's wait for @dblock for another pair of 👀.